wiz-admission-controller/templates/deployment.yaml

```diff
@@ -0,0 +1,99 @@
+      {{- end }}
+      labels:
+        {{- include "wiz-admission-controller.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "wiz-admission-controller.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+          - containerPort: {{ .Values.service.targetPort }}
+          volumeMounts:
+          - mountPath: /var/server-certs
+            name: server-certs
+            readOnly: true
+          readinessProbe:
+            httpGet:
+              path: /ready
+              port: 8000
+              scheme: HTTPS
+          livenessProbe:
+            httpGet:
+              path: /live
+              port: 8000
+              scheme: HTTPS
+          command:
+          - "/usr/bin/wiz-admission-controller-server"
+          - "--port={{ .Values.service.targetPort }}"
+          - "--tls-private-key-file=/var/server-certs/tls.key"
+          - "--tls-cert-file=/var/server-certs/tls.crt"
+          - "--error-enforcement-method={{ .Values.opaWebhook.errorEnforcementMethod }}"
+          - "--policy-enforcement-method={{ .Values.opaWebhook.policyEnforcementMethod }}"
+          {{- if .Values.opaWebhook.policies }}
+          - {{ include "wiz-admission-controller.opaCliParams.policies" . | trim | quote }}
+          {{- end }}
+          {{- if .Values.opaWebhook.clusterExternalId }}
+          - "--cluster-external-id={{ .Values.opaWebhook.clusterExternalId }}"
+          {{- end }}
+          env:
+          - name: WIZ_ADMISSION_CONTROLLER_CLIENT_ID
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "wiz-admission-controller.secretApiToken" . }}
+                key: clientId
+                optional: false
+          - name: WIZ_ADMISSION_CONTROLLER_CLIENT_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "wiz-admission-controller.secretApiToken" . }}
+                key: clientToken
+                optional: false
+          - name: WIZ_ADMISSION_CONTROLLER_ENV
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "wiz-admission-controller.secretApiToken" . }}
+                key: clientEnv
+                optional: false
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+      volumes:
+        - name: server-certs
+          secret:
+            defaultMode: 444
+            secretName: {{ include "wiz-admission-controller.fullname" . }}-cert
```

Comment:
Probably better to define this in the helpers and refer to it in both places that use it rather than rewrite this name in both places.